tests/llm/fixtures/test_ask_holmes/115_checkout_errors_tracing/traffic-generator.yaml

```diff
@@ -46,7 +46,7 @@ stringData:
         while True:
             request_count += 1
 
-            if request_count > 100 and not ever_had_promo:
+            if request_count > 20 and not ever_had_promo:
```

Comment:
I intentionally added a message after the first 100, not 20. why changing this?

Reply:
Fixing